hack/testing/util.sh

```diff
@@ -165,6 +165,17 @@ function add_test_message() {
        http://localhost:5601/$1 > /dev/null 2>&1
 }
 
+# $1 the timeout in seconds
+function flush_fluentd_pos_files() {
+    local timeout=${1:-60}
+    os::cmd::try_until_success "sudo rm -f /var/log/journal.pos" $(( timeout * second ))
```

Comment:
Why is this a `try_until_success`?  Is it because if fluentd is still running, it will keep trying to recreate the file until it is completely shutdown?

Reply:
I guess it doesn't need to be try_until_success... I made it so we only call this function when fluentd is stopped and before we start it again